sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -221,51 +240,37 @@ export abstract class LinkEntity<LinkT extends Receiver | AwaitableSender | Requ
 
     if (options.name) {
       this.name = options.name;
-      this._logPrefix = `[${connectionId}|${this._linkType}:${this.name}|a:${this.address}]`;
+      this._logPrefix = `[${connectionId}|${this._linkType}:${this.name}]`;
     }
 
     if (this._wasClosedPermanently) {
-      log.error(`${this._logPrefix} Link has been closed. Not reopening.`);
-      return;
+      log.error(`${this._logPrefix} Link has been permanently closed. Not reopening.`);
+      throw new Error(`Link has been permanently closed. Not reopening.`);
```

Comment:
Sorry if this is me being thick...
Previously, if _wasClosedPermanently was true after negotiating claim, we threw an AbortError, but here we are throwing a normal error. I recall you having very specific reasons to use the AbortError. The same does not apply here?

Reply:
No, that's a perfect fit, will change.

The only important thing was that it wasn't retryable.

Reply:
Changed over to AbortError.